src/components/recovery/CancelRecoveryButton/index.tsx

```diff
@@ -37,9 +39,11 @@ export function CancelRecoveryButton({
     e.preventDefault()
 
     if (isOwner) {
+      trackEvent(RECOVERY_EVENTS.CANCEL_RECOVERY)
```

Comment:
Why behind an if? A click is a click.

Reply:
It is also possible for non-owners to cancel the recovery tx after it has expired. Not sure if we should track that as a separate event or if both should be counted under the same event. cc @alicecha

Reply:
I've changed it for now so we track this event in both cases.